recipe/meta.yaml

```
@@ -1,34 +1,79 @@
+# Notes: this receipt is based on torchvision-feedstock: https://github.com/conda-forge/torchvision-feedstock/blob/master/recipe/meta.yaml
+
 {% set name = "torchani" %}
-{% set version = "2.2" %}
+{% set version = "2.2.1" %}
+{% set torchani_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c086fe6cabe49a0408d1f1b12aaf14e33b9e6ef0ffb15f3020792934e7c9fbbe
-  
+  git_url: https://github.com/aiqm/{{ name }}.git
+  git_rev: {{ version }}
+
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

```suggestion
  skip: true  # [win]
  skip: true  # [cuda_compiler_version == '10.2']
```

Lets just skip cuda 10.2

---

Hi, cuda 10.2 is still supported with `pytorch-gpu` for regular pytorch operators, the only problem here is our cuda_extension `cuaev` with not work with cuda 10.2 for now because of cub.

---

@yueyericardo `cub` is packaged on conda forge at https://github.com/conda-forge/cub-feedstock. That would probably be best to use this one. This is the one I used in my previous PR: https://github.com/hadim/torchani-feedstock/blob/cuda/recipe/meta.yaml#L43

---

thanks, let me try!

---

Hi actually conda-forge cub does not work, because pytorch does not include the conda include path.

We pined cub 1.11.0 at our setup.py for cuda 10.2: https://github.com/aiqm/torchani/pull/601
Cuda 11.0+ has cub builtin, so we don't need to worry about cub in the future.

So cuda 10.2 should working now

---

Ok, looks good to me.